Makefile

```diff
@@ -117,6 +121,11 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+openshift-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	VAL="${RH_RBAC_IMAGE}" yq e '.spec.template.spec.containers[0].image = strenv(VAL)' -i config/default/manager_auth_proxy_patch.yaml
```

Comment:
Is `yq` supposed to be `jq`? If so are we assuming it's installed by default?

Reply:
I think yq is to yaml as jq is to json

Reply:
Yes, yq is to yaml what jq is to json, exactly. I call out it's requirements in the description of this PR: `The openshift-deploy Make target requires yq to be installed (see https://github.com/mikefarah/yq). ` - I'll update the docs when the time comes but we have a separate task for that - also, it is expected that most people will still install the operator through the catalog and this Make target is probably going to be for development only, so I don't think it's unreasonable to add another dependency.

Reply:
I think I had the page open before you updated the description, so I didn't see it. Sounds good.